README.md

```diff
@@ -0,0 +1,69 @@
+## pivotal-tracker.rb - the original ruby gem
+
+This gem provides a wrapper around Version 5 of the Pivotal Tracker API
```

Comment:
Add a link to the API docs?  Bonus points for making "Pivotal Tracker API" the link...  `(https://www.pivotaltracker.com/help/api/rest/v5)`